### PR TITLE
Feature/nijo 502 update to133

### PR DESCRIPTION
**Pull request by mingazhev**

### CODE REVIEW

File: HappyTravel.EdoContracts/General/Price.cs [hunk 1]
```diff
@@ -23,6 +24,7 @@ public Price(Currencies currency, decimal netTotal, decimal gross, PriceTypes ty
         public string Description { get; }
         public decimal Gross { get; }
         public decimal NetTotal { get; }
+        public decimal? Discount { get; }
```
Comment (hasmikharutyunyan) [inline HappyTravel.EdoContracts/General/Price.cs:27]: We not need description for discount here? For what is discount?

Comment (mingazhev) [reply HappyTravel.EdoContracts/General/Price.cs:27]: I think this is to tell users that usual price is higher and now discount is big

Comment (mingazhev) [reply HappyTravel.EdoContracts/General/Price.cs:27]: Do you think that discount should be described right in the price?

Comment (hasmikharutyunyan) [reply HappyTravel.EdoContracts/General/Price.cs:27]: As I remember Illusions have description for discount, maybe we can add it to remarks?

Comment (mingazhev) [reply HappyTravel.EdoContracts/General/Price.cs:27]: I think such value should be added to the whole room contract bu this is discussable. @oleg1std @kirillta what do you think?

File: HappyTravel.EdoContracts/Accommodations/Internals/UniqueAccommodationCodes.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+using Newtonsoft.Json;
+
+namespace HappyTravel.EdoContracts.Accommodations.Internals
+{
+    public readonly struct UniqueAccommodationCodes
+    {
+        [JsonConstructor]
+        public UniqueAccommodationCodes(string? giataId = null)
```
Comment (kirillta) [inline HappyTravel.EdoContracts/Accommodations/Internals/UniqueAccommodationCodes.cs:8]: Alongside with a GIATA code we need our own one. And maybe it should be placed closer to an object root

Comment (mingazhev) [reply HappyTravel.EdoContracts/Accommodations/Internals/UniqueAccommodationCodes.cs:8]: HtId? PmsId?

File: HappyTravel.EdoContracts/General/Price.cs (hunk 1 above)
Comment (mingazhev) [reply HappyTravel.EdoContracts/General/Price.cs]: Added multiple discounts. Total discount can be calculated

File: HappyTravel.EdoContracts/Accommodations/Internals/UniqueAccommodationCodes.cs (hunk 2 above)
Comment (kirillta) [reply HappyTravel.EdoContracts/Accommodations/Internals/UniqueAccommodationCodes.cs:8]: Yes, something like that

Comment (mingazhev) [reply HappyTravel.EdoContracts/Accommodations/Internals/UniqueAccommodationCodes.cs:8]: Added HtId

**Pull request merged**